docs/library/Getting-Started.md

```diff
@@ -19,23 +19,23 @@ in the database.
 ## Installation
 
 To use the Nominatim library, you need access to a local Nominatim database.
-Follow the [installation and import instructions](../admin/) to set up your
-database.
+Follow the [installation](../../admin/Installation.md) and
+[import](../../admin/Import.md) instructions to set up your database.
```

Comment:
That's still not right. It has to be `(../admin/Installation.md)`. Interlinking in mkdocs is just painful.